content/en/platform/corda/5.0-dev-preview-2/getting-started/fast-feedback-with-the-simulator/fast-feedback-with-the-simulator.md

```diff
@@ -1,6 +1,6 @@
 ---
 date: '2022-09-20'
-title: "Fast feedback with Simulator"
+title: "Fast Feedback With Simulator"
```

Comment:
```suggestion
title: "Fast Feedback with Simulator"
```

Reply:
The common approach is that 'with' is capitalised, as it is 4 letters+ and doesn't fit into the other exclusion categories.